src/codegen/mod.rs

```diff
@@ -2062,6 +2077,13 @@ impl ToRustTy for Type {
                 let ident = ctx.rust_ident(&name);
                 quote_ty!(ctx.ext_cx(), $ident)
             }
+
+            TypeKind::ObjCInterface(..) => {
+                let name = item.canonical_name(ctx);
```

Comment:
This will be enough if you can't mix objective-c with namespaces. If you can, you need to handle them somehow (hint: you could use `utils::build_templated_path`, but pass always an empty `vec![]` of template arguments).